2nd-gen/time-server/README.md

```diff
@@ -0,0 +1,103 @@
+# Firebase SDK for Cloud Functions Quickstart - HTTPS trigger
+
+This quickstart demonstrates using the **Firebase SDK for Cloud Functions** with an HTTPS trigger through building an endpoint returning the current time.
+
+
+## Introduction
+
+The function `date` returns the current server date. You can pass it a `format` URL Query parameter to format the date.
+
+Further reading:
+
+ - [Read more about the Firebase SDK for Cloud Functions](https://firebase.google.com/docs/functions)
+
+
+## Initial setup, build tools and dependencies
```

Comment:
@egilmorez maybe we should pare this down so we're not duplicating docs? This probably applies to a lot of samples across the entire repo and we won't change all those now, but we can at least be consistent in v2 samples.

Reply:
This does seem lengthy . . . but I wouldn't want to do anything non-essential to slow the wheels of progress, and this feels like a nice-to-have.